chain-indexer/src/infra/subxt_node/runtimes.rs

```
@@ -185,13 +189,75 @@ async fn make_block_details_runtime_0_17(
                 )));
             }
 
+            // DUST registration events from NativeTokenObservation pallet.
+            Event::NativeTokenObservation(native_token_event) => match native_token_event {
+                NativeTokenObservationEvent::Registration(event) => {
+                    let cardano_address = CardanoStakeKey::from(event.cardano_address.0);
+                    if let Ok(dust_address_array) =
```

Why are errors swallowed? Shouldn't we err using `?`?
Applies to below, too.